### PR TITLE
feat(asset): display multiple asset for RGB++

### PR DESCRIPTION
**Pull request by Daryl-L**
https://github.com/Magickbase/ckb-explorer-public-issues/issues/658

https://ckb-explorer-frontend-in-magickbase-repo-git-a20841-magickbase.vercel.app/address/tb1qqlwxhg2jw392la2sesqqf4thjkz6q7nr4hcwwz

image

image

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -444,6 +444,9 @@
       "cell-info": "Cell Info"
     },
     "rgbpp": {
+      "assets": {
+        "multiple_warning": "The current UTXO contains multiple assets; please ensure that the assets are not lost when consuming the UTXO."
```
Comment (Keith-CY) [inline src/locales/en.json:448]:
```suggestion
        "multiple_warning": "The current UTXO contains multiple assets. Please ensure that the assets are not lost when consuming the UTXO."
```

File: src/locales/zh.json [hunk 2]
```diff
@@ -444,6 +444,9 @@
       "cell-info": "Cell详情"
     },
     "rgbpp": {
+      "assets": {
+        "multiple_warning": "当前UTXO下有多个资产，消耗UTXO时请确保资产不会丢失"
```
Comment (Keith-CY) [inline src/locales/zh.json:448]:
```suggestion
        "multiple_warning": "当前 UTXO 下有多个资产，消耗 UTXO 时请确保资产不会丢失"
```

**Review by Keith-CY — APPROVED** [review]

**Comment by FrederLu** [comment]
Request URL:  https://ckb-explorer-api-staging.magickbase.com/api/v1/addresses/tb1qdj3cv8jktgvl4ey4h7v24xuthc8ta99gj4a6pf
image

1、Compared with the API data, the data in the red box on the screenshot is abnormal data.

---
image

https://ckb-explorer-frontend-in-magickbase-repo-l26huxr04-magickbase.vercel.app/address/tb1qdj3cv8jktgvl4ey4h7v24xuthc8ta99gj4a6pf
image

2、In the UI, elements within a card should be left-aligned from top to bottom.

---
image
image

3、Compared with UI, the quantity does not need to be displayed. At the same time, when the mouse is hovered over `!`, the mouse style needs to be processed and prompt information needs to be displayed.

**Comment by FrederLu** [comment]
Request URL:
https://ckb-explorer-api-staging.magickbase.com/api/v2/bitcoin_addresses/tb1qdj3cv8jktgvl4ey4h7v24xuthc8ta99gj4a6pf/rgb_cells?page=1&page_size=10&sort=capacity.desc

image
API prompt 404.

**Comment by Daryl-L** [comment]
> Request URL: https://ckb-explorer-api-staging.magickbase.com/api/v2/bitcoin_addresses/tb1qdj3cv8jktgvl4ey4h7v24xuthc8ta99gj4a6pf/rgb_cells?page=1&page_size=10&sort=capacity.desc
> 
> image API prompt 404.

Could you please have a check? @rabbitz

**Comment by rabbitz** [comment]
> > Request URL: https://ckb-explorer-api-staging.magickbase.com/api/v2/bitcoin_addresses/tb1qdj3cv8jktgvl4ey4h7v24xuthc8ta99gj4a6pf/rgb_cells?page=1&page_size=10&sort=capacity.desc
> > image API prompt 404.
> 
> Could you please have a check? @rabbitz

The code became invalid due to conflicts on staging, but the issue has been resolved.

**Comment by FrederLu** [comment]
> > > Request URL: https://ckb-explorer-api-staging.magickbase.com/api/v2/bitcoin_addresses/tb1qdj3cv8jktgvl4ey4h7v24xuthc8ta99gj4a6pf/rgb_cells?page=1&page_size=10&sort=capacity.desc
> > > image API prompt 404.
> > 
> > 
> > Could you please have a check? @rabbitz
> 
> The code became invalid due to conflicts on staging, but the issue has been resolved.


Now the interface is normal, but there is a problem with the page style. @Daryl-L 
https://ckb-explorer-frontend-in-magickbase-repo-5y8xtqziq-magickbase.vercel.app/address/tb1qdj3cv8jktgvl4ey4h7v24xuthc8ta99gj4a6pf
image

**Comment by Daryl-L** [comment]
> > > Request URL: https://ckb-explorer-api-staging.magickbase.com/api/v2/bitcoin_addresses/tb1qdj3cv8jktgvl4ey4h7v24xuthc8ta99gj4a6pf/rgb_cells?page=1&page_size=10&sort=capacity.desc
> > > image API prompt 404.
> > 
> > 
> > Could you please have a check? @rabbitz
> 
> The code became invalid due to conflicts on staging, but the issue has been resolved.

Should respond as object, not the string. @rabbitz 
image

**Comment by rabbitz** [comment]
> > > > Request URL: https://ckb-explorer-api-staging.magickbase.com/api/v2/bitcoin_addresses/tb1qdj3cv8jktgvl4ey4h7v24xuthc8ta99gj4a6pf/rgb_cells?page=1&page_size=10&sort=capacity.desc
> > > > image API prompt 404.
> > > 
> > > 
> > > Could you please have a check? @rabbitz
> > 
> > 
> > The code became invalid due to conflicts on staging, but the issue has been resolved.
> 
> Should respond as object, not the string. @rabbitz image

It has been fixed.

**Comment by FrederLu** [comment]
https://ckb-explorer-api-staging.magickbase.com/api/v2/bitcoin_addresses/tb1qdj3cv8jktgvl4ey4h7v24xuthc8ta99gj4a6pf/rgb_cells?page=1&page_size=10&sort=capacity.desc
image
This interface request is not sorted, and the sorting information in the request can be removed.

**Comment by Daryl-L** [comment]
> /address/tb1qqlwxhg2jw392la2sesqqf4thjkz6q7nr4hcwwz

This rule was defined 3 months ago, I didn't change it.
image

**Pull request merged**